components/mpas-albany-landice/driver/glc_cpl_indices.F

```diff
@@ -14,25 +14,34 @@ module glc_cpl_indices
 
   ! Note that, in both the drv -> glc and the glc -> drv fields, index 0 means bare land
 
-! TODO: Comment these variables!
-
-  ! drv -> glc
-
-  integer, public :: index_x2g_Flgl_qice = 0 !Jer: removed multiple fields, now that these are confined to coupler
-  integer, public :: index_x2g_Fogo_qicel = 0 
-  integer, public :: index_x2g_Sl_tsrf   = 0 !Jer: removed multiple fields, now that these are confined to coupler
-  integer, public :: index_x2g_Fogo_qiceh = 0 !Jer:need to rename this everywhere...
```

Comment:
@JeremyFyke , what was x2g_Fogo_qiceh and why has it been removed/renamed?

Reply:
By the name, this was a heat flux field from coupler to glacier (originally from the ocean, I think).  My nominal guess here is that I renamed this to something else, or realized it was not needed.